### PR TITLE
[HttpFoundation] update doc creation session table

### PR DESCRIPTION
**Pull request by alli83**
linked to issue https://github.com/symfony/symfony-docs/issues/17569 and PR https://github.com/symfony/symfony/pull/49097/files

Update the methods to create the session table.

### CODE REVIEW

File: session.rst [hunk 1]
```diff
@@ -883,6 +935,10 @@ file and run the migration with the following command:
 
 .. _mysql:
```
Comment (stof) [inline session.rst]: this must stay just before the title.

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by stof** [comment]
this change is correct but it is not specific to databases. Passing a DSN as handler_id is supported in Symfony 4.4+ and supports other kinds of handlers too (if you pass a `redis://` URL, it will create a redis handler).
See https://github.com/symfony/symfony/pull/34177 for the PR implementing it and https://github.com/symfony/symfony-docs/pull/13227 for the PR documenting it.
So I'm not sure this should be repeated in this code snippet.

IMO, this PR should focus only on the issue it attempts to fix, i.e. documenting the integration with the doctrine schema tool.

**Comment by alli83** [comment]
@stof I see, thanks for your feedback ! I'll update it

**Comment by javiereguiluz** [comment]
@alli83 thanks a lot for improving the docs and congrats on your first Symfony Docs contribution 🎉 

Thanks also to Christophe for reviewing this.

**Pull request merged**